Examples/VaporExample/Package.swift

```diff
@@ -1,11 +1,14 @@
-// swift-tools-version:4.2
+// swift-tools-version:5.0
 import PackageDescription
 
 let package = Package(
     name: "VaporExample",
+    platforms: [
+       .macOS(.v10_14)
```

Comment:
requirement of Vapor 4

Reply:
Will this go away once it gets out of beta?

Reply:
actually, [they are planning to drop 10.14](https://forums.swift.org/t/announcement-vapor-4-will-no-longer-support-macos-10-14/33511/6) at some point before the release to use SwiftCrypto, so we'll probably have to update this to 10.15 once that happens.